integration/cf-compiler-test.ts

```diff
@@ -40,6 +40,13 @@ test.describe("cloudflare compiler", () => {
           import { content as browserPackage } from "browser-pkg";
           import { content as esmOnlyPackage } from "esm-only-pkg";
           import { content as cjsOnlyPackage } from "cjs-only-pkg";
+          import hooks from "node:async_hooks";
```

Comment:
@jplhomer looking good!

I think the last thing we need is to add a check in that same integration test to make sure node:async_hooks doesn't end up in the browser package. We've used shelljs's grep as a platform-agnostic way to check the browser assets, so you could do that here.

Reply:
Added a test for this! Granted, they aren't included because the import isn't being used in a browser context and thus is being stripped out as server code. Presumably, if the user attempts to import it into e.g. a React component, the bundle would break just like any other node external?